### PR TITLE
Add docs on Garbage Collector Admin View

### PR DESCRIPTION
**Pull request by codyjlandstrom**

### CODE REVIEW

File: src/content/admin/cleanup.mdx [hunk 1]
```diff
@@ -10,9 +10,38 @@ import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 import TiersList from "@theme/TiersList";
 
-The Okteto Garbage Collector helps keep your infra costs under control by sleeping or deleting inactive environments.
+The Okteto Garbage Collector (GC) allows Admins to optimize infrastructure usage by automatically scaling down or deleting inactive Developer and Preview environments, helping control costs and improve resource efficiency.
 
-## Sleeping Resources
+## Enabling the Garbage Collector
+
+To enable the GC, navigate to **Admin → Garbage Collector** and click `Install` if it’s not already installed.
```
Comment (viddo) [inline src/content/admin/cleanup.mdx:17]: This is not correct. "Install" CTA just defers to these same docs here. Or should we link the install step to some other docs? (would require changing app code though)

@teresaromero @pchico83 How should we explain required actions an admin needs to take? How does one _uninstall/install_ GC exactly? I think this is related to the [Helm chart configuration](https://www.okteto.com/docs/1.26/self-hosted/helm-configuration/#gc) that I also think need a review and more consideration for the new feature, right? Also see/review [current changes](https://github.com/okteto/docs/pull/898/files#diff-5246fa871de346de64f5cbaf3735699dde71a326365ff143ce7219a80f38f839) below.

Comment (codyjlandstrom) [reply src/content/admin/cleanup.mdx:17]: Ok, thanks. I was basing this off the screenshots that showed the "Install" button without realizing that it just linked to the docs. I can edit the docs to reflect this, but we should also update the helm configuration to describe "installation"

File: src/content/admin/dashboard.mdx [hunk 2]
```diff
@@ -207,4 +209,17 @@ If you choose to pull images from a private registry in addition to [Okteto's Co
 
 Dynamically configure Okteto's connection to AWS and GCP, eliminating the need for developers to manage static credentials.
 
-[Review this page for a full description of how to configure and start using Cloud Credentials →](admin/cloud-credentials/index.mdx)
\ No newline at end of file
+[Review this page for a full description of how to configure and start using Cloud Credentials →](admin/cloud-credentials/index.mdx)
+
+## Garbage Collector
+
+Configure sleep and delete periods for Okteto Developer Environments and Preview Environments from the UI.
```
Comment (pchico83) [inline src/content/admin/dashboard.mdx:216]: Same as before, today, "Developer Environments" is not used across product/docs

**Review by pchico83 — APPROVED** [review]

File: src/content/admin/dashboard.mdx (hunk 2 above)
Comment (codyjlandstrom) [reply src/content/admin/dashboard.mdx:216]: Thanks, will update both of these

**Pull request merged**